setup.py

```diff
@@ -19,18 +19,18 @@
 version = "1.17.0"
 release_status = 'Development Status :: 3 - Alpha'
 dependencies = [
-    "requests",
-    "attrs",
+    "requests>=2.31.0",
+    "attrs>=20.1.0",
     "click>=8.0.4, <8.1.0",
-    "cryptography",
-    "google-auth",
-    "jinja2",
-    "keyring",
-    "packaging",
-    "protobuf>=3.19.4, <4.0.0",
-    "pyjwt",
-    "pyperclip",
-    "python-dateutil",
+    "cryptography>=42",
+    "google-auth>=1.10.1",
+    "jinja2>=3.1.3",
+    "keyring>=21.8.0",
+    "packaging>=20.0",
+    "protobuf>=4.21.6",
```

Comment:
Can we separate out the protobuf major version bump into a separate PR? This looks like a dangerous change (for potentially breaking the triggering of our builds) and we don't have great tests around it.

If you are confident that the wire format is unchanged, then perhaps we can merge together.

Reply:
> If you are confident that the wire format is unchanged, then perhaps we can merge together.

Perhaps we should publish an rc version with these changes and test it out. WDYT?